Makefile

```diff
@@ -44,7 +44,8 @@ CFLAGS += -Wall
 CFLAGS += -Wextra
 CFLAGS += -Werror
 CFLAGS += -pedantic -Wpedantic
-CLFAGS += -Wno-cast-function-type -Wno-error=cast-function-type
+# needed in 20.04
+#CFLAGS += -Wno-cast-function-type -Wno-error=cast-function-type
```

Comment:
Is there a way to fix this for 20.04 without removing these flags?
I believe there should be some warnings that pop up during compilation.

Reply:
Sorry little misleading here. There's actually no effect for this change here. This flag was never effective due to previous typo(was **CLFAGS** instead of **CFLAGS**). I added it back in 20.04 with typo fix but it won't work for 18.04 so commented it out.